packages/dataviews/src/view-table.js

```diff
@@ -94,49 +83,45 @@ function HeaderMenu( { field, view, onChangeView } ) {
 					) }
 				</Button>
 			}
+			style={ { minWidth: '240px' } }
 		>
 			<WithSeparators>
 				{ isSortable && (
 					<DropdownMenuGroup>
-						{ Object.entries( sortingItemsInfo ).map(
+						{ Object.entries( SORTING_DIRECTIONS ).map(
 							( [ direction, info ] ) => {
-								const isActive =
+								const isChecked =
 									isSorted &&
 									view.sort.direction === direction;
 								return (
-									<DropdownMenuItem
+									<DropdownMenuRadioItemCustom
```

Comment:
Same comment as https://github.com/WordPress/gutenberg/pull/55625/files#r1437560831

Why do we need a custom radio here? Since the sorting semantics changed at https://github.com/WordPress/gutenberg/pull/56717 we don't need to "uncheck" a checked option.

Reply:
Using normal radios would originally cause the bug discussed in [this comment](https://github.com/WordPress/gutenberg/pull/55625#issuecomment-1864772748) for the sorting options

> > In terms of functionality I spotted a small issue. If you sort by one field, then sort by another, the original sorting remains selected in the menu. See the screenshot below where I sorted by title, then by author. The menu tells me the data is still sorted by title when it is not
> 
> That is a consequence of using "real" menu radio items — the value in the radio group can not be reset to having no items selected.
> 
> For the time being, I [extracted a "custom menu radio item" component](https://github.com/WordPress/gutenberg/pull/55625/commits/771313ff8f2f82e4f0744905465d78ddf852d7cc) based on the existing code, and reverted all menu items to using the custom implementation which should allow for deselecting items ([`86748ca` (#55625)](https://github.com/WordPress/gutenberg/pull/55625/commits/86748ca5889ab4c56bceb6f0c53e7a0ca12053ac)).


But I think I found a way to still use the actual radio component, will put up a PR soon.